### PR TITLE
[UXD][AAP-25027] AWX - updates to Projects section breadcrumbs & headers

### PR DESCRIPTION
**Pull request by tiyiprh**
https://issues.redhat.com/browse/AAP-25027

Updates to Projects section areas breadcrumbs and headers

### CODE REVIEW

File: cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe('Project Edit, Copy, Sync', () => {
       cy.get(`[data-cy="row-id-${project.id}"]`).within(() => {
         cy.get('[data-cy="edit-project"]').click();
       });
-      cy.verifyPageTitle('Edit Project');
+      cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```
Comment (ZitaNemeckova) [inline cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts:34]: @tiyiprh This name is not correct. As names in tests are generated it cannot be hardcoded like here. Instead something like 
```
cy.verifyPageTitle(`Edit ${project.name}`);
```

File: cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts [hunk 2]
```diff
@@ -40,31 +40,31 @@ describe('Project Edit, Copy, Sync', () => {
         .its('response.body')
         .then((edited: Project) => {
           expect(edited.name).to.eql(`${project.name} - edited`);
-          cy.verifyPageTitle(`${edited.name}`);
+          cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```
Comment (ZitaNemeckova) [inline cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts:43]: This is detail page that only has project name. This change is not needed.

File: cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts [hunk 3]
```diff
@@ -40,31 +40,31 @@ describe('Project Edit, Copy, Sync', () => {
...
           cy.clickButton(/^Edit project$/);
           cy.get('[data-cy="name"]').clear().type(`${project.name}`);
           cy.clickButton(/^Save project$/);
-          cy.verifyPageTitle(project.name);
+          cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```
Comment (ZitaNemeckova) [inline cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts:47]: This is detail page that only has project name. This change is not needed.

File: cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts [hunk 4]
```diff
@@ -40,31 +40,31 @@ describe('Project Edit, Copy, Sync', () => {
...
         });
     });
 
     it('can edit a project from the project details page', () => {
       cy.clickTableRowLink('name', project.name, { disableFilter: true });
-      cy.verifyPageTitle(project.name);
+      cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```
Comment (ZitaNemeckova) [inline cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts:53]: This is detail page that only has project name. This change is not needed.

File: cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts [hunk 5]
```diff
@@ -40,31 +40,31 @@ describe('Project Edit, Copy, Sync', () => {
...
       cy.clickButton(/^Edit project$/);
-      cy.verifyPageTitle('Edit Project');
+      cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```
Comment (ZitaNemeckova) [inline cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts:55]: This name is not correct. As names in tests are generated it cannot be hardcoded like here. Instead something like
```
cy.verifyPageTitle(`Edit ${project.name}`);
```

File: cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts [hunk 6]
```diff
@@ -40,31 +40,31 @@ describe('Project Edit, Copy, Sync', () => {
...
       cy.get('[data-cy="name"]').clear().type(`${project.name} - edited`);
       cy.intercept('PATCH', awxAPI`/projects/${project.id.toString()}/`).as('edited');
       cy.clickButton(/^Save project$/);
       cy.wait('@edited')
         .its('response.body')
         .then((edited: Project) => {
           expect(edited.name).of.eql(`${project.name} - edited`);
-          cy.verifyPageTitle(`${project.name} - edited`);
+          cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```
Comment (ZitaNemeckova) [inline cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts:63]: This is detail page that only has project name. This change is not needed.

File: cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts [hunk 7]
```diff
@@ -40,31 +40,31 @@ describe('Project Edit, Copy, Sync', () => {
...
           cy.clickButton(/^Edit project$/);
           cy.get('[data-cy="name"]').clear().type(`${project.name}`);
           cy.clickButton(/^Save project$/);
-          cy.verifyPageTitle(project.name);
+          cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```
Comment (ZitaNemeckova) [inline cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts:67]: This is detail page that only has project name. This change is not needed.

File: frontend/awx/resources/projects/ProjectPage/ProjectForm.cy.tsx [hunk 8]
```diff
@@ -140,7 +140,7 @@ describe('ProjectForm.cy.ts', () => {
     );
     cy.intercept('PATCH', 'api/v2/projects/*', { statusCode: 201 }).as('editProject');
     cy.mount(<EditProject />);
-    cy.verifyPageTitle('Edit Project');
+    cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```
Comment (nixocio) [inline frontend/awx/resources/projects/ProjectPage/ProjectForm.cy.tsx:143]: Coming from the fixtures file.

File: frontend/awx/resources/projects/ProjectPage/ProjectForm.tsx [hunk 9]
```diff
@@ -128,10 +128,16 @@ export function EditProject() {
   return (
     <PageLayout>
       <PageHeader
-        title={t('Edit Project')}
+        title={
+          project?.name ? t('Edit {{projectName}}', { projectName: project?.name }) : t('Project')
```
Comment (nixocio) [inline frontend/awx/resources/projects/ProjectPage/ProjectForm.tsx:132]: Should not be lower case the last project?

Comment (nixocio) [reply frontend/awx/resources/projects/ProjectPage/ProjectForm.tsx:132]: It is a fallback there is not name for a project, but name is a required field.

**Review by nixocio — APPROVED** [review]

**Pull request merged**